oke_cluster_engine.tf

```diff
@@ -27,6 +27,13 @@ locals {
   is_flexible_node_shape = contains(local.compute_flexible_shapes, var.oke_node_shape)
 }
 
+data "template_file" "oke_node_cloudinit_content" {
```

Comment:
This is to allow the OKE nodes to run a command to use all the space assigned to the boot volume, so the pods won't run out of space.